src/crimson/os/seastore/logging.h

```diff
@@ -12,23 +12,23 @@
 
 #ifdef NDEBUG
 
-#define LOG(LEVEL, MSG, ...) LOGGER . LEVEL("{}: " MSG, FNAME __VA_OPT__(,) __VA_ARGS__)
-#define LOGT(LEVEL, MSG, t, ...) LOGGER . LEVEL("{}({}): " MSG, FNAME, (void*)&t __VA_OPT__(,) __VA_ARGS__)
+#define LOG(level_, MSG, ...) LOGGER.log(level_, "{}: " MSG, FNAME __VA_OPT__(,) __VA_ARGS__))
+#define LOGT(level_, MSG, t, ...) LOGGER.log(level_, "{}({}): " MSG, FNAME, (void*)&t __VA_OPT__(,) __VA_ARGS__))
 
-#define TRACE(...) LOG(trace, __VA_ARGS__)
-#define TRACET(...) LOGT(trace, __VA_ARGS__)
+#define TRACE(...) LOG(seastar::log_level::trace, __VA_ARGS__)
+#define TRACET(...) LOGT(seastar::log_level::trace, __VA_ARGS__)
 
```

Comment:
can we just do "using namespace seastar::log_level" to avoid adding prefix to each line.

Reply:
i'd like to expose less names in global naming space.